# EntitySrv.msg
uint32 agentId
float64 beginTime
float64 durationTime
string taskCode
Multipath paths   